.github/workflows/install-valkey/action.yml

```diff
@@ -0,0 +1,70 @@
+name: Install Valkey
+
+inputs:
+    valkey-version:
+        description: "valkey version to install"
+        required: true
+        type: string
+    target:
+        description: "Specified target toolchain, ex. x86_64-unknown-linux-gnu"
+        type: string
+        required: true
+        options:
+            - x86_64-unknown-linux-gnu
+            - aarch64-unknown-linux-gnu
+            - x86_64-apple-darwin
+            - aarch64-apple-darwin
+            - aarch64-unknown-linux-musl
+            - x86_64-unknown-linux-musl
```

Comment:
This used as a cache key, so could be anything. This parameter and options looks like copied from rust toolchain selector.
I think you can remove this parameter and use combination of "os/runner + version" as a cache key